solr/distribution/build.gradle

```diff
@@ -59,11 +59,10 @@ configurations {
 
 dependencies {
   changesHtml project(path: ":solr:documentation", configuration: "changesHtml")
-  docker project(path: ':solr:docker', configuration: project.ext.withSignedArtifacts ? 'packagingOfficial' : 'packaging')
+  docker project(path: ':solr:docker', configuration: project.ext.withSignedArtifacts ? 'packagingOfficial' : 'packagingLocal')
```

Comment:
@HoustonPutman I added a new configuration `packagingLocal` since the existing `packaging` configuration did not produce the `docker/` output. This is kind of a quick fix, it may also be possible to change the `from(configurations.docker` below to pull out `Dockerfile.local` from the result of `packaging`?

Reply:
Ahhh I understand why this broke now, because we renamed `Dockerfile.local` to `Dockerfile` in the binary release. This looks good to me.